src/components/my_meeting/HomeMyMeetings.js

```diff
@@ -1,15 +1,38 @@
 "use-strict";
 import React, { Component } from "react";
-import { View, Text } from "react-native";
+import { View, StyleSheet, FlatList, Dimensions } from "react-native";
+import { CRHeader } from "src/components/common"
+import MeetingCell from "src/components/my_meeting/MeetingCell"
+
+const cellWidth = Dimensions.get("window").width;
+const data = [{ id: "0", title: "hola" }, { id: "1", title: "Hola" }, { id: "2", title: "Hola" }, { id: "3", title: "Hola" }];
 
 class HomeMyMeetings extends Component {
   render() {
     return (
-      <View>
-        <Text>FHomeMyMeetings Screen</Text>
+      <View style={styles.main}>
+        <CRHeader leftText="Profile" />
+        <FlatList
+          data={data}
+          renderItem={this.renderItem}
+          keyExtractor={item => item.id}
+        />
       </View>
     );
   }
+
+  renderItem = ({ item }) => (
```

Comment:
favor components over render methods

Reply:
What do you mean? All examples I've seen of how using FlatList are in this format.

Reply:
Create a component instead of using methods:

```jsx
const Item = ({ onPressAppoint, title }) => (
  <View style={{ width: cellWidth, height: cellWidth * 0.5, padding: 5 }}>
    <MeetingCell onPress={onPressAppoint} text={title} />
  </View>
);
```

Reply:
Got it, thanks !

Reply:
done